### DISCUSSION TITLE
Add IGV tutorial

### DISCUSSION THREAD

**Discussion by aklin2** [sandbox-bio/sandbox.bio#5] [Q&A] [answered]
There are a couple of sections that ask to view a genomic region in IGV.
So it would be nice to have a tutorial on how to use IGV or a tutorial on a similar tool.

**Comment by robertaboukhalil** [comment, marked as answer]
@aklin2 Just published an IGV tutorial! See https://sandbox.bio/tutorials?id=igv-intro

**Reply by aklin2** [reply]
OOoooh sick, will give it a try, thanks!!! This must have taken a lot of work, I appreciate the updates :)
